float32 front_distance
float32 left_distance
float32 right_distance
float32 back_distance